pyproject.toml

```diff
@@ -0,0 +1,60 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [ "setuptools>=45", "setuptools_scm[toml]>=6.2",]
+
+[project]
+classifiers = [ "Development Status :: 2 - Pre-Alpha", "Natural Language :: English", "Programming Language :: Python :: 3",]
```

Comment:
Typhos shouldn't be pre-alpha I think

Reply:
Probably [Development Status :: 5 - Production/Stable](https://pypi.org/search/?c=Development+Status+%3A%3A+5+-+Production%2FStable)?

Reply:
I think so, I should have googled the options myself

Reply:
No worries.  That hyperlink does not go where I thought it did